File: fizzbuzz/fizzbuzz.cpp
#include <iostream>
#include <string>

using std::cout;
using std::endl;
using std::string;

int main() {
  for (int i = 1; i < 101; i++) {
    string s = "";
    if (i % 3 == 0) {
      s += "Fizz";
    }
    if (i % 5 == 0) {
      s += "Buzz";
    }
    if (s.length() == 0) {
      cout << i << endl;
    } else {
      cout << s << endl;
    }
  }
}

File: fizzbuzz/fizzbuzz.py
#!/usr/bin/python

for i in xrange(1,101):
  s = ""
  if i % 3 == 0:
    s += "Fizz"
  if i % 5 == 0:
    s += "Buzz"
  print i if len(s) == 0 else s
